pxb/percona-xtrabackup-8.0-template.yml

```diff
@@ -41,8 +41,8 @@
                 PKGLIST+=" libcurl-devel cmake libaio-devel zlib-devel libev-devel bison make gcc"
                 PKGLIST+=" rpm-build libgcrypt-devel ncurses-devel readline-devel openssl-devel gcc-c++"
                 PKGLIST+=" vim-common rpmlint wget ncurses-compat-libs python2 python2-setuptools python2-wheel"
-                PKGLIST+=" lz4 lz4-devel"
-                until yum -y install ${PKGLIST}; do
+                PKGLIST+=" lz4 lz4-devel perl-DBD-MySQL"
```

Comment:
do we need it for deb based systems?

Reply:
It's already there: https://github.com/Percona-Lab/jenkins-pipelines/blob/f2fa5dc78cd83e28bce4fd56ebd2e617eae5f544/pxb/percona-xtrabackup-8.0-template.yml#L128